tests/ignite/metrics/regression/test_fractional_absolute_error.py

```diff
@@ -28,14 +28,15 @@ def test_wrong_input_shapes():
         m.update((torch.rand(4, 1), torch.rand(4)))
 
 
-def test_compute():
+def test_compute(available_device):
     a = np.random.randn(4)
     b = np.random.randn(4)
     c = np.random.randn(4)
     d = np.random.randn(4)
     ground_truth = np.random.randn(4)
 
-    m = FractionalAbsoluteError()
+    m = FractionalAbsoluteError(device=available_device)
```

Comment:
Let's rewrite this test: let's create a, b, c, d variables with torch tensor API instead of numpy